README.md

```diff
@@ -30,9 +30,7 @@ This is the recommended way to include SwiftLint in your workflow, as it catches
 
 Alternately, a SwiftLint scheme is exposed within the project; Xcode will show a warning if you don't have SwiftLint installed.
 
-Finally, you can also run SwiftLint manually from the command line with:
-
-`rake lint`
+SwiftLint is connected to Simplenote iOS, so lint errors will appear as warnings in Xcode after you build the project.
```

Comment:
How about something in the lines of:

```
SwiftLint is integrated directly into the Xcode project, so lint errors appear as warnings after you build the project
```

SwiftLint connected to Simplenote iOS feels like it's doing something with the actual service maybe?

Reply:
Sounds good, have made that change